management-gateway/src/main/java/nl/nn/adapterframework/util/JacksonUtils.java

```diff
@@ -51,13 +51,13 @@ public static <T> T convertToDTO(Object payload, Class<T> dto) {
 				return MAPPER.readValue((String) payload, dto);
 			} else if(payload instanceof byte[]) {
 				return MAPPER.readValue((byte[]) payload, dto);
+			} else {
+				throw new NotImplementedException("unhandled payload type ["+payload.getClass()+"]");
```

Comment:
Potential NPE here? (Perhaps not a big issue since it's a throw exception either way).